app-office/libreoffice/libreoffice-4.4.9999.ebuild

```diff
@@ -9,7 +9,7 @@ QT_MINIMAL="4.7.4"
 KDE_SCM="git"
 CMAKE_REQUIRED="never"
 
-PYTHON_COMPAT=( python2_7 python3_3 python3_4 )
+PYTHON_COMPAT=( python{2_7,3_3,3_4} )
```

Comment:
Nope, we don't drop stuff retroactively unless it's already masked or otherwise broken.